### DISCUSSION TITLE
Using abbreviations which conflict with those from the standard

### DISCUSSION THREAD

**Discussion by hardreddata** [australiangeomechanics/agsformat#69] [Q&A] [answered]
Hi,

Thanks for the great work in preparing and maintaining this standard.

Suppose companies are interested in using abbreviations which conflict with those from the standard. For historical reasons people may be familiar with a particular shorthand.

Consider the DISC_APOB heading where the standard (Excel workbook) stipulates code "CN" with description "Clean". I guess one could use code "CLN" for this and still adhere to the standard, provided any AGS file includes the definition? Similarly for DISC_PLAN one might have a preference for "PL" rather than "PR" to indicate "Planar".

I am asking about changing the codes on existing terms rather than adding entirely new terms with new meanings.

Is this all fine?

Many thanks.

**Comment by SolidRockProgrammer** [comment, marked as answer]
The AGS file must use the abbreviations that are defined in AGS 4.1.1 AU if one exists for that thing.  You can use new abbreviations if there isn't an existing code for that thing.

This is a data interchange format, so everyone needs to be talking the same language at the time of interchange.  You can use other codes in your internal processes, just convert them when you export AGS format data.
